docs/values.rst

```diff
@@ -120,6 +120,21 @@ instead of using the name of the :class:`~Value` instance.::
     class Dev(Configuration):
         TIME_ZONE = values.Value('UTC', environ_name='MYSITE_TZ')
 
+Allow final value to be used outside the configuration context
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Sorry, there seems to be a problem with the font config in Qutebrowser for me:
![2017-07-16-160832_663x350_scrot](https://user-images.githubusercontent.com/9766/28248219-1d34f092-6a41-11e7-96e8-92101b2125e4.png)

It looks fine in Firefox, so sorry for the confusion.